README.md

```diff
@@ -17,8 +17,8 @@ The original version of this tool was created by [New Relic](https://github.com/
     # Defaults to $GITHUB_WORKSPACE.
     directory: ''
 
-    # A path to the JSON/YAML Repolinter ruleset to use, relative to the workflow
-    # working directory (i.e. under `$GITHUB_WORKSPACE`).
+    # A path to the JSON/YAML Repolinter ruleset to use, relative to the
+    # `directory` input.
```

Comment:
This is technically a breaking change? But it shouldn't be an issue because no one uses the `directory` parameter IIRC.

Reply:
So, I was wondering about that too... is anyone *actually* using this in a way that is going to break.  I included the `BREAKING CHANGE` text in my commit, which I think will cause semantic-release to bump the major version.  But do we actually want that here?

Reply:
Naw definitely not, it'll create a new major version tag and we'll have to switch to `todogroup/repolinter-action@v2`. Maybe tag it as a feature or a fix?

My bad for not reading the extended description btw, I forgot it existed.